flag_test.go

```diff
@@ -356,3 +356,33 @@ func TestNoInterspersed(t *testing.T) {
 		t.Fatal("expected interspersed options/non-options to fail")
 	}
 }
+
+func TestTermination(t *testing.T) {
+	f := NewFlagSet("termination", ContinueOnError)
+	if f.Parsed() {
+		t.Error("f.Parse() = true before Parse")
+	}
+	arg1 := "ls"
+	arg2 := "/home"
```

Comment:
This actually will fail without the fix, as the arguments are doubled (`ls /home ls /home`). In this case, `len(f.Args())` is 4 without the fix. But I have switch to `ls -l` and adding a `-l` bool flag to verify that it's not set. There is also some overlap of testing termination and flags in `TestShorthand` as well.